app/services/standardize_address_service.rb

```diff
@@ -0,0 +1,52 @@
+require 'net/http'
+require 'uri'
+
+# PLEASE do NOT use this service to standardize addresses in bulk (do NOT use in backfills)
+# as that violates the terms of agreement with USPS and will result in access being revoked.
+class StandardizeAddressService
+  def self.standardize_address_on_intake(intake)
+    build_standardized_address(intake.street_address, intake.city, intake.state, intake.zip_code)
+  end
+
+  private
+
+  def self.build_standardized_address(street_address, city, state, zip_code)
+    usps_address_xml = get_usps_address_xml(street_address, city, state, zip_code)
+    {
+      street_address: usps_address_xml.xpath("//Address/Address2").text,
```

Comment:
My PREFERENCE would be to create instances of this class and be able to get the response as method calls, but that's just a preference and not backed up by any specific need. However, it would allow us to normalize when there are error responses maybe?